src/Tribe/Event_Repository.php

```diff
@@ -46,7 +95,7 @@ public function __construct() {
 	 *                                        `NOT BETWEEN` operators without passing a two element array `$value`.
 	 */
 	public function filter_by_cost( $value, $operator = '=', $symbol = null ) {
-		if ( ! in_array( $operator, array(
+		if ( ! in_array( $operator, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 4c03cc5fe98e735533aeab24897d3faa</sub>